components/navbar/sidebar.snip.html

```diff
@@ -17,11 +17,14 @@
 }}
 
 <!-- Start Sidebar -->
-<amp-sidebar id="{{sidebar-id}}" class="ampstart-sidebar px3 {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}}" layout="nodisplay">
+<amp-sidebar id="{{sidebar-id}}" class="ampstart-sidebar px3 {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}} {{sidebarclasses}}" layout="nodisplay">
```

Comment:
Does this mean we shouldn't hardcode any classes into the template itself? Changing this file would require changing almost all pages within ampstart (at least 16), would it be possible to file this as a bug and do it in a follow up PR?